src/components/common/Button.css.ts

```diff
@@ -0,0 +1,99 @@
+import { vars } from '@/theme/globalTheme.css';
+import { recipe } from '@vanilla-extract/recipes';
+
+const pressedButtonStyle = (color: string) => ({
+  selectors: {
+    '&:hover:not(:disabled)': {
+      color: color,
+    },
+    '&:active:not(:disabled)': {
+      color: color,
+    },
+    '&:focus:not(:disabled)': {
+      color: color,
+    },
+  },
+});
+
+export const buttonStyle = recipe({
+  base: [
+    {
+      borderWidth: 0,
+      borderRadius: 100,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      color: vars.color.white100,
+      width: '100%',
+      marginBottom: 24,
+
+      ':disabled': {
+        backgroundColor: vars.color.black08,
+        color: vars.color.black40,
+      },
+    },
+    pressedButtonStyle(vars.color.white64),
+  ],
+  variants: {
+    type: {
+      primary: {
+        backgroundColor: vars.color.black100,
+      },
+      secondary: {
+        backgroundColor: vars.themeColor.color.secondary100,
```

Comment:
```suggestion
        backgroundColor: vars.color.secondary100,
```
컬러토큰을 동일한 방식으로 쓰는게 좋을 것 같아서 vars.themeColor를 color로 통합시켰어용
혹시 vars.themeColor를 사용하는 코드가 있으면 다 제거 부탁드립니닷!
관련 pr: #5

Reply:
@hayoiii 
다행이도(?) 여기만 사용하고 있네요 > [refactor: themecolor -> color로 통합](https://github.com/preznto/frontend/pull/6/commits/5cfb25fea18cd1e2205f3c0573fb69b254f73fe0)